llvm/test/tools/lto/discard-value-names.ll

```diff
@@ -7,11 +7,10 @@
 
 ; The test requires asserts, as it depends on the default value for
 ; -lto-discard-value-names at the moment.
-; FIXME: -lto-discard-value-names is ignored at the moment.
 
 ; REQUIRES: asserts
```

Comment:
Should the REQUIRED to be removed?

Reply:
I think it is needed as otherwise the default is to discard value names. Left as is for now